docs/user_guide/learn_more.md

```diff
@@ -22,20 +22,22 @@ The following blog posts have previously been published about Venice:
 - 2022: [Open Sourcing Venice – LinkedIn’s Derived Data Platform](https://engineering.linkedin.com/blog/2022/open-sourcing-venice--linkedin-s-derived-data-platform)
 - 2023: [Public CI](https://blog.venicedb.org/public-ci)
 - 2023: [Stable Releases](https://blog.venicedb.org/stable-releases)
+- 2024: [VeniceCon 2024](https://blog.venicedb.org/venicecon-2024)
+- 2024: [Stable Release 0.4.263](https://blog.venicedb.org/stable-release-0-4-263)
 
 Follow us on [HashNode](http://blog.venicedb.org) or [subscribe to the newsletter](https://blog.venicedb.org/newsletter) 
 to hear about it when new posts are published.
 
 ## Talks
 The following talks have been given about Venice:
 
-- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE)
+- 2018: [Venice with Apache Kafka & Samza](https://www.youtube.com/watch?v=Usz8E4S-hZE) # TODO: private video?
```

Comment:
Looks like that video is gone indeed... let's comment out the whole line. We can always uncomment it later if we figure out how to get to it again.